model/colorspace.go

```diff
@@ -2401,32 +2400,36 @@ func (cs *PdfColorspaceSpecialIndexed) ImageToRGB(img Image) (Image, error) {
 	baseImage.Height = img.Height
 	baseImage.Width = img.Width
 	baseImage.alphaData = img.alphaData
-	baseImage.BitsPerComponent = img.BitsPerComponent
+	baseImage.BitsPerComponent = 8 // cs.Base.BitsPerComponent
```

Comment:
Please remove commented code.  Was there a specific reason for using 8 explicitly than the commented out `cs.Base.BitsPerComponent` ?

Would be good to add a comment referencing https://github.com/unidoc/unipdf/issues/260  that more bit depth support is needed (TODO).

Apparently this is not a regression? Technically the `img.GetSamples` should work for arbitrary bitdepth.
I guess we do need a test suite for other bit depths.

We should also add a document that showcases the need for this fix into the test corpus.  The file is already provided in the referencing issue so that should be straightforward to add to the hash table.

Reply:
I added the TODO. 
I am not sure if `cs.Base.ImageToRGB` will work for cs.Base bit depths other than 8. At some stage I can instrument this code, run it over the PaperCut corpus and see if there are any indexed color spaces over base color space with bit depths other than 8. However  PaperCut has reporting any bugs from customer other than the one in 258 so I suspect this is not a common case.
https://github.com/unidoc/unipdf/issues/258 has the PDF document that shows the need for the fix